### PR TITLE
Laghos: new package

### PR DESCRIPTION
**Pull request by bhatele**
cc @goxberry

### CODE REVIEW

File: var/spack/repos/builtin/packages/mfem/package.py [hunk 1]
```diff
@@ -112,6 +115,7 @@ class Mfem(Package):
     depends_on('lapack', when='+suite-sparse')
 
     depends_on('mpi', when='+mpi')
+    depends_on('hypre', when='+hypre')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/mfem/package.py]: it's already there, several lines above.

**Comment by bhatele** [comment]
@davydden where's the typo?

**Comment by davydden** [comment]
> where's the typo?

i mean extra line, added accidentally, see above

**Comment by bhatele** [comment]
@davydden Fixed.

**Pull request merged**